start_server.py

```
@@ -0,0 +1,93 @@
+#!/usr/bin/env python3
+"""
+Start the ch11.ai API server with controlled output
+"""
+
+import subprocess
+import time
+import signal
+import sys
+import json
+
+def signal_handler(sig, frame):
```

Ruff (0.11.9)

Missing return type annotation for public function `signal_handler`

Add return type annotation: `None`

(ANN201)